governance_blueprint/validation/generate_artifact_manifest.py

```
@@ -151,6 +157,15 @@ def main() -> int:
             return 1
         expected_obj = build_manifest(preserve_timestamp=True)
+        if current_obj.get("package") != expected_obj.get("package"):
+            print("artifact_manifest.json has mismatched package metadata")
+            return 1
+        if current_obj.get("version") != expected_obj.get("version"):
+            print("artifact_manifest.json has mismatched version metadata")
+            return 1
+        current_artifacts = current_obj.get("artifacts", {})
+        expected_artifacts = expected_obj.get("artifacts", {})
+        if current_artifacts != expected_artifacts:
         if current_obj != expected_obj:
```

Pylint (4.0.5)

[error] Parsing failed: 'expected an indented block after 'if' statement on line 168 (generate_artifact_manifest, line 169)'

(E0001)